core/src/avm2/property_map.rs

```diff
@@ -66,10 +66,13 @@ impl<'gc, V> PropertyMap<'gc, V> {
     }
 
     pub fn get_for_multiname(&self, name: &Multiname<'gc>) -> Option<&V> {
+        if name.has_lazy_component() {
+            panic!("Bug: lookup on uninitialized Multiname");
```

Comment:
Can you include the multinsme in the panic message? Ritht now, this panic will be pretty uninformstive if it actually happens.

Reply:
Sure (tbh now that I think about it, this should be `unreachable!()`, as the only way for this to be currently reached is via an invariant break in `Activation`)

Reply:
(that's why I didn't think that multiname text matters, as this is more about the code than the AS or name itself)